Create Github PR template (#152)
steelthreads/agds-next: .github/pull_request_template.md

## Describe your changes
Include photos if necessary.

## Checklist

> - [ ] Run `yarn format` over your changes

Reviewer: ```suggestion
- [ ] Run `yarn format`
```

> - [ ] Ensure `yarn lint` and `yarn tsc` checks are passing

Reviewer: ```suggestion
- [ ] Run `yarn lint` in the root of the repository to ensure tests are passing
```

Reviewer: You only need to run `yarn lint` as that command runs `yarn lint:typescript`

- [ ] Add necessary tests

> - [ ] Add Changeset

Reviewer: ```suggestion
- [ ] Run `yarn changeset` to create a  changeset file
```

> - [ ] Write documentation (README.md)

Reviewer: Should these 2 items be moved under For new components... ?

Author: Changesets and documentation should also be implemented for any component updates :)
